09-features.md

```diff
@@ -29,7 +29,12 @@ These flags may only be used in the `init` message:
 
 ## Assigned `globalfeatures` flags
 
-There are currently no `globalfeatures` flags.
+These flags may only be used in the `init` and `node_announcement` message:
+
+| Bits | Name             |Description                                     | Link                                                                |
+|------|------------------|------------------------------------------------|---------------------------------------------------------------------|
+| 16/17 | `option_support_large_channel` | Can create large channels		   | [Rationale](#Rationale) |
```

Comment:
Why start at 16/17 instead of zero? The existing local feature bits followed the pattern of always starting new features from the lowest unassigned value, which in this case is zero.

Reply:
Because it took in account the feature bits unification. I agree that if we aren't going to merge #571 we should start the `globalfeatures` from zero but **if** we later merge it then we need to update this feature bits (localfeature 0/1 is already defined), in that scenario is better to start from 16/17 to avoid collisions.